packages/contracts-bedrock/.storage-layout

```diff
@@ -49,15 +47,15 @@
 +---------------+-------------------------------------------------+------+--------+-------+----------------------------------------------------+
 | Name          | Type                                            | Slot | Offset | Bytes | Contract                                           |
 +==============================================================================================================================================+
-| messenger     | contract CrossDomainMessenger                   | 0    | 0      | 20    | contracts/L1/L1StandardBridge.sol:L1StandardBridge |
+| _initialized  | uint8                                           | 0    | 0      | 1     | contracts/L1/L1StandardBridge.sol:L1StandardBridge |
```

Comment:
Storage layout changed, but it's OK. We'll need to review all of the storage layouts before we go to mainnet. I spoke with @tynes and we agreed that it would be better to make storage layout changes now for the sake of readability/speed for now and make tweaks to match the original storage layout at the very end of this process.